firebase-firestore/ktx/src/test/kotlin/com/google/firebase/firestore/ktx/JavaLibKtxSupportTests.kt

```diff
@@ -0,0 +1,195 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.firestore.ktx
+
+import com.google.common.truth.Truth.assertThat
+import com.google.firebase.Timestamp
+import com.google.firebase.firestore.DocumentId
+import com.google.firebase.firestore.DocumentReference
+import com.google.firebase.firestore.GeoPoint
+import com.google.firebase.firestore.IgnoreExtraProperties
+import com.google.firebase.firestore.ServerTimestamp
+import com.google.firebase.firestore.ThrowOnExtraProperties
+import com.google.firebase.firestore.documentReference
+import java.util.Date
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import org.junit.Test
+
+class JavaLibKtxSupportTests {
+
+    @Test
+    fun `java converted Ktx ServerTimestamp annotations should be seen during Ktx serialization`() {
```

Comment:
```suggestion
    fun `ServerTimestamp annotations should be seen during Ktx serialization`() {
```

Reply:
You can remove that prefix from your test name since the conversion is a one time process, after that they are just the "regular" annotations.